wcivf/apps/people/models.py

```diff
@@ -223,6 +223,14 @@ def statement_remainder(self):
 
         return statement_remainder
 
+    @property
+    def deceased(self):
```

Comment:
I think it's useful to abstract this, but It might need a slightly different name - for example, imagine we want to show a list of deceased candidates in future (let's not do this!), we might be tempted to iterate over candidates and do `if person.deceased`. That makes sense, but we might not realise that it's actually checking other things too.

Can we call this....er...`should_show_deceased_text`? Is that what it's for?

Reply:
Sorry you tidied this up later - glad you caught this!

Reply:
In a later version of this file, we call this method `display_deceased`. Ok?

Reply:
Absolutely - sorry this is one of the things I missed the later change to as I was working though the commits